src/Utils/Rules.js

```diff
@@ -0,0 +1,48 @@
+const Rules = {
+  isNotRangedValueInNumbers(numbers) {
+    return numbers.some((number) => number < 1 || number > 45);
+  },
```

Comment:
```suggestion
  isNotRangedValueInNumbers(numbers) {
    return numbers.some((number) => isNotRangedValue(number));
  },
```
아래에서 정의한 `isNotRangedValue` 메서드를 사용해도 좋을 것 같아요!